zero Add: create .txt file with metadata

MayaAbe opened the pull request:
地上局から`b'm'`を送信したときに，`./ImageJPG/`内にある
`.jpg`ファイルのメタデータを書き込んだ同名の`.txt`ファイル
を生成する機能を`./scsat1-cam`に追加した．

> 例）
> (実行前)
> ```
> ~/ImageJPG $ ls
> image1.jpg image2.jpg
> ```
> 
> (実行後)
> ```
> ~/ImageJPG $ ls
> image1.jpg image1.txt image2.jpg image2.txt
> ```


処理が正常に行われたときには`b'm'`が返され，
エラーが生じたときには`b'e'`が返される．

現在エラー内容は`print()`を用いて表示されるが，
デバッグ用途であるため後に取り除く予定．

Commit:
zero: rpios: Add text file creation with metadata

地上局から`b'm'`を送信したときに，`./ImageJPG/`内にある
`.jpg`ファイルのメタデータを書き込んだ同名の`.txt`ファイル
を生成する機能を`./scsat1-cam`に追加した．

処理が正常に行われたときには`b'm'`が返され，
エラーが生じたときには`b'e'`が返される．

現在エラー内容は`print()`を用いて表示されるが，
デバッグ用途であるため後に取り除く予定．

Signed-off-by: MayaAbe <a.maya.3141592@icloud.com>
Signed-off-by: Takuya Sasaki <takuya.sasaki@spacecubics.com>

sasataku commented:
Commitメッセージの `.txt` は、`text` でいいかな。
(拡張子が大事なのではなく、text fileに書くこと大事なので)

あと、タイトルは
`zero: rspi: Add 〜`　にしてください。

sasataku commented on zero/rpios/scsat1-cam:
```diff
@@ -14,11 +15,22 @@ while True:
+            exifrw.meta_save_directory()
+        except Exception as e:
+            sock.sendto(b'e', addr)
+            print('Capture Error')
```
`Capture error` ではないので、適切なエラーメッセージに変更しましょう。

Commit:
zero: rpios: Fix comment and binary expression

１．'c'を受け取ったときの処理を説明する
　　コメントを追加
２．'e'を受け取ったときの処理を説明する
　　コメントの位置を修正
３．'e'を受け取ったときのレスポンスを
　　バイナリで返すように修正

Signed-off-by: MayaAbe <a.maya.3141592@icloud.com>
